css/src/atomics/overflow.scss

```diff
@@ -0,0 +1,29 @@
+$overflow-values: auto hidden;
+
+@each $value in $overflow-values {
```

Comment:
Is the looping needed? Thinking we'd might only need overflow-hidden. The classes defined in scroll.scss are what we'd want to direct developers towards.

Reply:
You are right, just updated